addons/frag/functions/fnc_addBlackList.sqf

```diff
@@ -1,21 +1,29 @@
 #include "..\script_component.hpp"
 /*
- * Author: Jaynus, NouberNou
- * Adds a round to the blacklist (will be ignored).
+ * Author: Jaynus, NouberNou, Lambda.Tiger
+ * Adds a round type to the blacklist (will be ignored).
  *
  * Arguments:
- * 0: Projectile <OBJECT>
+ * 0: Projectile <OBJECT> or classname <STRING>
  *
  * Return Value:
  * None
  *
  * Example:
- * [bullet] call ace_frag_fnc_addBlackList
+ * [projectile] call ace_frag_fnc_addBlackList
  *
  * Public: No
  */
 
-params ["_round"];
+params ["_proj"];
 TRACE_1("addBlackList",_round);
 
-GVAR(blackList) pushBack _round;
+switch (typeName _proj) do {
+    case "OBJECT": {
+        GVAR(shouldFragCache) set [typeOf _proj, false];
+    };
+    case "STRING": {
+        GVAR(shouldFragCache) set [_proj, false];
+    };
+    default { };
+};
```

Comment:
```suggestion
};

```

Newlines at end of file.

Reply:
Sorry about that, debug code is a long running personal code base and I forgot to update all the files. Two questions on formatting though:

1. Is `_proj` not verbose enough? I thought I'd seen it used in other places in ACE.
2. I didn't see it in the guidelines, but are hanging newlines required/desired?

Reply:
Yeah, newlines at the end of all files. Not sure why it's not in the guidelines but it's done everywhere else in ACE

Reply:
Ok!

Reply:
Should be good with trailing newlines as of https://github.com/acemod/ACE3/pull/9728/commits/a0812371c97c5d5eafb3f02e2af7bd689da5c0f8